CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 - Add `file` command
 
+- Upgrade Ruby (2.5.0)
```

Comment:
Since this will be for next release, I think these entries should be added before `## 2.7.3` under something like `## unreleased`.

Then, when running `release.sh` it will open this for edit, then we change `## unreleased` to `## 2.8.0`